doc/source/dev/toolchain.rst

```diff
@@ -108,7 +108,7 @@ Currently, SciPy wheels are being built as follows:
 Linux (nightly)    ``ubuntu-18.04``          GCC 6.5                      See ``azure-pipelines.yml``
 Linux (release)    ``ubuntu-18.04``          GCC 7.5                      Built in separate repo [6]_
 OSX                ``macOS-10.15``           LLVM 12.0.0                  Built in separate repo [6]_
-Windows            ``windows-latest``        Visual Studio 2019 (16.11)   See ``azure-pipelines.yml``
+Windows            ``windows-2019``          Visual Studio 2019 (16.11)   See ``azure-pipelines.yml``
```

Comment:
This is actually wrong - I overlooked that windows was built by appveyor also on the scipy-wheels repo. Thanks @charris!

Waiting for some further input in the discussion [here](https://github.com/numpy/numpy/pull/20524#issuecomment-987502235), placing this as a "blocker" in the meantime.